stdlib/sys/file.gr

```diff
@@ -298,7 +298,7 @@ let fdStats = (fd) => {
 
   let mut rightsInheritingList = []
   List.forEachi((flag, i) => {
-    if (rightsInheriting && Int64.lsl(1L, i) > 0L) {
+    if ((rightsInheriting && Int64.lsl(1L, i)) > 0L) {
```

Comment:
(`&&` is shadowed here to be `&` for Int64s.)

This also makes me think that even if they aren't defined, we should still parse `&` & friends so they can be defined in a context like this.

Reply:
https://github.com/grain-lang/grain/issues/297

Reply:
half-lol @ "Good First Issue"